pfio/CMakeLists.txt

```diff
@@ -92,7 +92,7 @@ set (srcs
   )
 
 esma_add_library (${this} SRCS ${srcs} DEPENDENCIES MAPL.shared MAPL.profiler NetCDF::NetCDF_Fortran TYPE ${MAPL_LIBRARY_TYPE})
-target_link_libraries (${this} PUBLIC GFTL_SHARED::gftl-shared PRIVATE MPI::MPI_Fortran)
+target_link_libraries (${this} PUBLIC esmf GFTL_SHARED::gftl-shared PRIVATE MPI::MPI_Fortran)
```

Comment:
There is ESMF in pFIO? That is surprising. Are we using some utility from it?

Reply:
__RC__ need "MAPL_Exceptions.h" which includes "ESMC_ReturnCodes.h"

Reply:
I agree - but maybe this got relaxed somewhere.   Really prefer for pfio to be independent of ESMF.